quixstreams/topic_manager.py

```diff
@@ -0,0 +1,756 @@
+import logging
+import warnings
+import pprint
+from abc import abstractmethod
+from typing import Dict, List, Mapping, Optional, Set, Literal, Protocol, ClassVar
+
+from quixstreams.platforms.quix import QuixKafkaConfigsBuilder
+from quixstreams.utils.dicts import dict_values
+from .kafka.admin import Admin
+from .models.serializers import DeserializerType, SerializerType
+from .models.topics import Topic, TopicConfig, TopicList, TopicMap
+
+logger = logging.getLogger(__name__)
+
+__all__ = ("TopicManager",)
+
+
+class BytesTopic(Topic):
+    def __init__(self, name: str, config: Optional[TopicConfig] = None):
+        super().__init__(
+            name=name,
+            key_serializer="bytes",
```

Comment:
What is the purpose of this class? 
It doesn't seem to be different from the `Topic` class, but it still represents a different type.